src/coreclr/src/binder/assemblybinder.cpp

```diff
@@ -380,23 +379,29 @@ namespace BINDER_SPACE
 
         _ASSERTE(ppSystemAssembly != NULL);
 
-        StackSString sCoreLibDir(systemDirectory);
         ReleaseHolder<Assembly> pSystemAssembly;
 
+        StackSString sCoreLibDir(systemDirectory);
         if (!sCoreLibDir.EndsWith(DIRECTORY_SEPARATOR_CHAR_W))
         {
             sCoreLibDir.Append(DIRECTORY_SEPARATOR_CHAR_W);
         }
 
-        StackSString sCoreLib;
+        StackSString sCoreLib(sCoreLibDir);
+        StackSString coreLibName(CoreLibName_IL_W);
+        sCoreLib.Append(coreLibName);
```

Comment:
Is this fake path going to be actually used for anything? Would it be better to not build it and pass it in at all when we find the payload in the bundle?